service/controller/v5/resource/blobobject/resource.go

```diff
@@ -0,0 +1,71 @@
+package blobobject
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-07-01/storage"
+	"github.com/giantswarm/azure-operator/client"
```

Comment:
Let's make sure we stick to `fmt`. Please also check other places where this is not aligned. 

```suggestion

	"github.com/giantswarm/azure-operator/client"
```